src/mesh/tools.rs

```diff
@@ -790,12 +790,13 @@ mod tests {
             Point3::origin(),
```

Comment:
If changes to this test happened only for debugging purposes, let's discard these changes.